elementary/monitor/data_monitoring/report/slack_report_summary_message_builder.py

```diff
@@ -146,6 +146,8 @@ def _add_details_to_slack_alert(
                 failed_tests_details.extend(
                     self._get_test_result_details_block(test, include_description)
                 )
+            elif test.status == "pass":
```

Comment:
I think it will be better to replace the elif pass with elif warning and the rest will be skipped :)